README.md

```diff
@@ -79,6 +79,11 @@ To destroy stack just type:
 ``~ $ perun delete-stack <NAME OF THE STACK>
 ``
 
+To create temporary secure credentials with MFA type:
+
+```bash
+~ $ perun mfa
```

Comment:
We've got separate section `Supporting MFA` - put it there. And add an information that you do not need to use Perun for validation, you can just use Perun to obtain secure credentials and use it in aws cli.